subt/docker/rospy/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:20.04
```

Comment:
Are you sure, that ROS for Ubuntu 18 (Melodic) and Ubuntu 20 (Noetic) are compatible?! (for some older versions this used to be headache)